core/common/src/main/java/alluxio/util/ConfigurationUtils.java

```diff
@@ -457,33 +463,51 @@ public static AlluxioConfiguration merge(AlluxioConfiguration conf, Map<?, ?> pr
   }
 
   /**
-   * Loads cluster default values from the meta master.
+   * Sends a request to meta master to set a property for a path pattern.
    *
-   * @param address the master address
-   * @param conf configuration to use
-   * @return a configuration object containing the original configuration merged with cluster
-   *         defaults, or the original object if the cluster defaults have already been loaded
+   * @param address the meta master address
+   * @param conf the cluster level configuration
+   * @param path the path pattern
+   * @param key the property key
+   * @param value the property value
    */
-  public static AlluxioConfiguration loadClusterDefaults(InetSocketAddress address,
-      AlluxioConfiguration conf)
-      throws AlluxioStatusException {
-    if (!conf.getBoolean(PropertyKey.USER_CONF_CLUSTER_DEFAULT_ENABLED)
-        || conf.clusterDefaultsLoaded()) {
-      return conf;
+  public static void setPathConfiguration(InetSocketAddress address, AlluxioConfiguration conf,
```

Comment:
this belongs in a `MetaMasterConfigurationClient`, similar to how we handle all other grpc clients, e.g. `MetaMasterClient`, `FileSystemMasterClient`, etc

Reply:
I agree, there is no existing `MetaMasterConfigurationClient` though, I'll create one.